tests/assert-jobs/utils/elasticsearch/elasticsearch.go

```diff
@@ -221,7 +221,7 @@ func executeEsRequest(es EsConnection, httpMethod, api string, body []byte) ([]b
 			Certificates: []tls.Certificate{es.Certificate},
 			RootCAs:      es.RootCAs,
 		}
-		tlsConfig.BuildNameToCertificate()
+
```

Comment:
What is the effect of this change?

Reply:
It seems in newer versions is not needed anymore. I ran all the E2E tests in OpenShift and they passed